app/Http/Controllers/Admin/Dashboard/AdminDashboardController.php

```diff
@@ -2,15 +2,41 @@
 
 namespace App\Http\Controllers\Admin\Dashboard;
 
+use App\Models\MenteeProfile;
+use App\Models\MentorProfile;
 use Inertia\Inertia;
 
 class AdminDashboardController
 {
     public function index()
     {
+        $mentors = [];
+        $mentorProfiles = MentorProfile::all();
+        foreach ($mentorProfiles as $m) {
+            $attr = $m->getAttributes();
+            $u = $m->user;
+            $attr['first_name'] = $u->first_name;
```

Comment:
I guess we could change these to map to camelCase keys here:
```
            $attr['firstName'] = $u->first_name;
```
etc.

Is this the best place to do this mapping?

Reply:
Yes, I think it is.